desktop/cmp/grouping/GroupingChooser.ts

```diff
@@ -87,45 +84,37 @@ export const [GroupingChooser, groupingChooser] = hoistCmp.withFactory<GroupingC
                 isOpen,
                 popoverRef: model.popoverRef,
                 popoverClassName: 'xh-grouping-chooser-popover xh-popup--framed',
-                // right align favorites popover to match star icon
-                // left align editor to keep in place when button changing size when commitOnChange: true
-                position: favoritesIsOpen ? `${popoverPosition}-right` : `${popoverPosition}-left`,
-                minimal: styleButtonAsInput,
+                // Left align editor to keep in place when button changing size when commitOnChange: true
+                position: `${popoverPosition}-left`,
+                minimal: false,
                 item: fragment(
                     button({
                         text: label,
                         title: label,
                         tabIndex: -1,
                         className: classNames(
                             'xh-grouping-chooser-button',
-                            styleButtonAsInput ? 'xh-grouping-chooser-button--as-input' : null,
-                            persistFavorites ? 'xh-grouping-chooser-button--with-favorites' : null
+                            styleButtonAsInput ? 'xh-grouping-chooser-button--as-input' : null
                         ),
                         minimal: styleButtonAsInput,
                         ...buttonProps,
                         onClick: () => model.toggleEditor(),
                         testId
-                    }),
-                    favoritesIconCmp({testId: favoritesIconTestId, favoritesIcon})
+                    })
                 ),
-                content: favoritesIsOpen
-                    ? favoritesMenu({testId: favoritesMenuTestId})
-                    : editor({
-                          popoverWidth,
-                          popoverMinHeight,
-                          popoverTitle,
-                          emptyText,
-                          testId: editorTestId
-                      }),
+                content: popoverCmp({
+                    popoverWidth,
+                    popoverMinHeight,
+                    popoverTitle,
+                    emptyText,
+                    testId
+                }),
                 onInteraction: (nextOpenState, e) => {
                     if (
                         isOpen &&
                         nextOpenState === false &&
                         e?.target &&
-                        !elemWithin(
-                            e.target as HTMLElement,
-                            'xh-grouping-chooser-button--with-favorites'
-                        )
+                        !elemWithin(e.target as HTMLElement, 'xh-grouping-chooser-button')
```

Comment:
This was only checking `'xh-grouping-chooser-button--with-favorites'` before but I don't know why that would be the case.  I want to understand/comment why we need this - as far as I can tell, it lets you...click the main button again to close the popover _without_ saving?  I didn't get it.....